pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -84,27 +66,27 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		//
 		// Only do this when doing a bit of management...
 		logger.Infof("creating %q configmap", k8sutil.ConfigOverrideName)
-		err = populateConfigOverrideConfigMap(c.context, c.namespacedName.Namespace, cluster.ownerRef)
+		err = populateConfigOverrideConfigMap(c.context, c.namespacedName.Namespace, cluster.ClusterInfo.OwnerRef)
 		if err != nil {
 			return errors.Wrap(err, "failed to populate config override config map")
 		}
 
 		logger.Infof("creating %q secret", config.StoreName)
-		err = config.GetStore(c.context, c.namespacedName.Namespace, &cluster.ownerRef).CreateOrUpdate(cluster.Info)
+		err = config.GetStore(c.context, c.namespacedName.Namespace, &cluster.ClusterInfo.OwnerRef).CreateOrUpdate(cluster.ClusterInfo)
 		if err != nil {
 			return errors.Wrap(err, "failed to update the global config")
 		}
 	}
 
 	// The cluster Identity must be established at this point
-	if !cluster.Info.IsInitialized(true) {
+	if !cluster.ClusterInfo.IsInitialized(true) {
 		return errors.New("the cluster identity was not established")
 	}
 	logger.Info("external cluster identity established")
 
 	// Create CSI Secrets only if the user has provided the admin key
-	if cluster.Info.AdminSecret != mon.AdminSecretName {
-		err = csi.CreateCSISecrets(c.context, c.namespacedName.Namespace, &cluster.ownerRef)
+	if cluster.ClusterInfo.CephCred.Username == client.AdminUsername {
+		err = csi.CreateCSISecrets(c.context, cluster.ClusterInfo, &cluster.ownerRef)
```

Comment:
Do we still need `&cluster.ownerRef` since we have `cluster.ClusterInfo.OwnerRef`?

Reply:
Good catch, I missed this duplicate ref.